programs/mango-v4/src/instructions/token_edit.rs

```diff
@@ -49,6 +49,7 @@ pub fn token_edit(
     maint_weight_shift_asset_target_opt: Option<f32>,
     maint_weight_shift_liab_target_opt: Option<f32>,
     maint_weight_shift_abort: bool,
+    fallback_oracle_opt: Option<Pubkey>,
```

Comment:
This passes the oracle twice, as an account and as an instruction argument. Doesn't seem to be necessary. Instead just pass a `set_fallback_oracle: bool`? (we already should have caught this for the main oracle)